### PR TITLE
Feature/merge to v2 drop legacy

### LINKED ISSUE #2321: [feature request] Merge admin and adminv2
**Is your feature request related to a problem? Please describe.**

There are two admin areas:

* Legacy – http://localhost:3000/admin
* New – http://localhost:3000/admin_v2

**Describe the solution you'd like**

These should be consolidated into a single admin area.
* Ideally, ensure all of the functionality from legacy admin (`/admin`) is available in the new admin (`/admin_v2`).
* New admin routes are moved to exist on the `/admin` URL

### PR DESCRIPTION
**Pull request by CDeighton**
PR Checklist

- [x] - Commit [messages are descriptive](https://github.com/ONEARMY/community-platform/blob/master/CONTRIBUTING.md#--commit-style-guide), it will be used in our [Release Notes](https://github.com/ONEARMY/community-platform/releases/)

PR Type

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [x] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Developer experience (improves developer workflows for contributing to the project)

## Description

This depends on #2463 and #2465

Drops the old `/admin` code now that all of the features are in `admin_v2`

## Git Issues

Closes #2321

### CODE REVIEW

File: src/modules/index.ts [hunk 1]
```diff
@@ -8,14 +8,14 @@ export enum MODULE {
...
-  ADMIN = 'admin_v2',
+  ADMIN = 'admin',
 }
 
 export const getSupportedModules = (): MODULE[] => {
   const envModules: string[] =
     getConfigurationOption(
       'REACT_APP_SUPPORTED_MODULES',
-      'howto,map,events,research,academy,user,admin_v2',
+      'howto,map,events,research,academy,user,admin',
```
Comment (CDeighton) [inline src/modules/index.ts:18]: Drops legacy admin and shifts `admin_v2` routes to `admin`

**Review by iSCJT — CHANGES_REQUESTED** [review]
@CDeighton one additional consideration with this is that the beta testers admin functionality isn't yet replicated in the new users page so that will also need to be replicated somewhere before fully deprecating v1 admin.

**Comment by AlfonsoGhislieri** [comment]
Going to assign this over to you @iSCJT!

**Comment by CDeighton** [comment]
> @CDeighton one additional consideration with this is that the beta testers admin functionality isn't yet replicated in the new users page so that will also need to be replicated somewhere before fully deprecating v1 admin.

@iSCJT oh yikes sorry 🤦 good catch! I thought the new `/admin_v2/users` route covered it but it looks like there is no way to add roles to users on that page.

If it is just the roles, how do you feel about having them on the `u/${username}/edit` screen (mockup below) - 

Screenshot 2023-06-28 at 11 29 37 PM

I'd hide the `super-admin` checkbox if a non-super-admin was looking at the screen of course. This way feels a little better to me because it consolidates the user management into a single place. Though it is very possible that I've missed another detail somewhere 😅 

I'm happy to just lift the existing screens as they are if you prefer though, I think it's roughly the same amount of work either way

**Comment by iSCJT** [comment]
@CDeighton No problem at all, it's not the most obvious and intuitively you'd expect it to be in the new users screen.  At the moment we're trying to centralise the admin functionality into one place rather than have functionality spread across different pages so just lift the existing page into the v2 admin please.  Thank you 🌟

**Comment by CDeighton** [comment]
@iSCJT should be good now 🤞 I'd recommend reviewing it commit by commit. Theres one for each of the admin and beta tester screens, then one to drop the old admin screen

**Pull request merged**